src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php

```diff
@@ -45,23 +48,50 @@ public function &Execute()
             }
 
             foreach (array_keys($aList) as $index) {
-                $oBreadCrum->AddItem($aList[$index]);
+                $breadcrumb->AddItem($aList[$index]);
             }
 
-            $this->data['oBreadcrumb'] = &$oBreadCrum;
+            $this->data['oBreadcrumb'] = &$breadcrumb;
```

Comment:
if this code is being changed anyway, you might as well remove the `&`

Reply:
ok